client/components/auth/user.service.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+angular.module('badgerApp')
+  .factory('User', function ($resource) {
+    return $resource('/api/users/:id/:controller', {
+      id: '@_id'
```

Comment:
A TODO comment should be added here to specify that a controller parameter needs to be added in the future.